tests/unit/sendATFReport.unitTest.ts

```diff
@@ -38,7 +42,7 @@ describe("sendATFReport", () => {
           vin: "XMGDE02FS0H012345",
           vehicleType: "psv",
         },
-      ],
+      ] as unknown as TestResultSchema[],
```

Comment:
Can we remove the `as unknown` and do this as just `as TestResultSchema[]` ? Same for anywhere else we're enforcing types